src/renderer/mesh_storage.zig

```diff
@@ -85,27 +85,27 @@ pub fn deinit() void {
 	for(mapStorageLists) |mapStorageList| {
 		main.globalAllocator.destroy(mapStorageList);
 	}
+
 	for(updatableList.items) |mesh| {
 		mesh.decreaseRefCount();
 	}
-	updatableList.deinit();
+	updatableList.clearAndFree();
 	for(mapUpdatableList.items) |map| {
 		map.decreaseRefCount();
 	}
-	mapUpdatableList.deinit();
+	mapUpdatableList.clearAndFree();
 	for(priorityMeshUpdateList.items) |mesh| {
 		mesh.decreaseRefCount();
 	}
-	priorityMeshUpdateList.deinit();
-	blockUpdateList.deinit();
-	meshList.deinit();
+	priorityMeshUpdateList.clearAndFree();
+	blockUpdateList.clearAndFree();
+	meshList.clearAndFree();
 	for(clearList.items) |mesh| {
 		mesh.deinit();
 		main.globalAllocator.destroy(mesh);
 	}
-	clearList.deinit();
+	clearList.clearAndFree();
 }
-
```

Comment:
Unintended line removal?

Reply:
It's an empty line, does it matter?

Reply:
No, just looked like it was unintentionally removed. If you made this change intentionally then I would like to know the intention though.

Reply:
ah i remember, that was when i made the "reset" function, but then we decided to just make the already existing deinit function use clearAndFree() , those removing the need for the reset function
while deleting the reset() function, i also deleted one line to much